backend/benefit/companies/api/v1/views.py

```diff
@@ -3,48 +3,55 @@
 from django.conf import settings
 from django.db import transaction
 from django.http import HttpRequest
-from drf_spectacular.utils import extend_schema, OpenApiParameter, OpenApiTypes
+from django.utils.translation import gettext_lazy as _g
```

Comment:
No biggie, but I think in other parts of the codebase the gettext_lazy is imported and used just as  a "_"

Reply:
`_` was already in the scope of some function where "defined but not used" -warning needed to be silenced by naming some function argument as `_`. I will see if there is another way to deal with this.

Reply:
> `_` was already in the scope of some function where "defined but not used" -warning needed to be silenced by naming some function argument as `_`. I will see if there is another way to deal with this.

Maybe `__` xD